Source/FullyConnectedFunctions/arm_fully_connected_s4.c

```diff
@@ -21,8 +21,8 @@
  * Title:        arm_fully_connected_s4
  * Description:  Fully connected function compatible with TF Lite.
  *
- * $Date:        10 October 2023
- * $Revision:    V.1.0.0
+ * $Date:        22 April 2024
+ * $Revision:    V.1.1.0
```

Comment:
Should this be v2?

Reply:
No I would say since the prototype of arm_fully_connected_s4 is not changed, just the implementation

Reply:
Ah my bad, the cropping of the file confused me, thought this was prototype!